.github/actions/matrix-fan-out/action.yml

```diff
@@ -0,0 +1,33 @@
+name: "Matrix Fan-out"
+description: "Uploads a JSON file as an artifact with a unique name."
+
+inputs:
+  path:
+    description: "The directory to upload."
+    default: ./matrix-fan-out
+  retention-days:
+    description: "Optional. Defaults to repository settings."
+    default: null
+  shell:
+    default: bash -euxo pipefail {0}
```

Comment:
Missing description which is required according to GitHub's schema (https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddescription)

Reply:
The selftest passes, so it doesn't seem to be very required, to me.

Reply:
dup